components/HeaderDesktop.tsx

```diff
@@ -11,45 +11,40 @@ import FrenchFlag from './svg/flags/FrenchFlag'
 import GermanFlag from './svg/flags/GermanFlag'
 import { calcRem } from '../utils/styles'
 import { SectionProps } from './sections/utils/SectionProps'
-import { ReactNode } from 'react'
+import React, { ReactNode, useState } from 'react'
 
 export function HeaderDesktop({ className }: SectionProps) {
   return (
     <div
-      className={classNames(
-        'hidden laptop:flex justify-between w-full px-14 py-4 absolute',
-        className
-      )}
+      className={classNames('hidden w-full laptop:grid py-4 pl-8', className)}
+      style={{
+        gridTemplateColumns: 'auto auto'
+      }}
     >
       <Link href="/">
         <a>
           <img src="/logo/polarfox.png" alt="Polarfox logo" />
         </a>
       </Link>
       <div
-        className="flex items-center space-x-20 font-semibold"
+        className="grid grid-flow-col items-center justify-end text-center font-semibold"
         style={{
           fontSize: calcRem(14),
           lineHeight: calcRem(16.45)
         }}
       >
         <ProductsDropdown />
-        {/* TODO: Remove the div below */}
-        <div />
         <Link href="/about">
-          <a>About</a>
+          <a className="px-6">About</a>
```

Comment:
Idea: refactor code to remove `px-6` and use `gap-x-6` or `space-x-6` instead (or `12`)

Reply:
I don't see how i can change this because padding need to be applied only on <Link>

Reply:
Just tried it on my machine.

You can add `gap-x-14`, remove `px-6` here, and remove `px-6` at line 75 (in the `Dropdown()` function). This produces a result that is very similar.